packages/eslint-plugin/typings/eslint-rules.d.ts

```diff
@@ -759,6 +759,7 @@ declare module 'eslint/lib/rules/lines-around-comment' {
         allowModuleEnd?: boolean;
         ignorePattern?: string;
         applyDefaultIgnorePatterns?: boolean;
+        afterHashbangComment?: boolean;
```

Comment:
I am not sure if our additional options should belong here – when I first read through this I thought base ESlint rule supports options like `allowInterfaceStart`. IMO only base options should belong here and we should extend them in the rule file. WDYT?

Reply:
Agreed, feels off for that reason to put them here.